articles/machine-learning/how-to-machine-learning-interpretability-aml.md

```diff
@@ -37,10 +37,13 @@ For more information on the supported interpretability techniques and machine le
 ## Generate feature importance value on your personal machine 
 The following example shows how to use the interpretability package on your personal machine without contacting Azure services.
 
-1. Install `azureml-interpret` and `azureml-contrib-interpret` packages.
+1. Install `azureml-interpret`, `azureml-contrib-interpret` and other required packages.
     ```bash
     pip install azureml-interpret
     pip install azureml-contrib-interpret
+    
+    pip install interpret-community[visualization]
+    pip install -r https://raw.githubusercontent.com/interpretml/interpret-community/master/requirements.txt --user
```

Comment:
I don't think this should be required for all users:
```
    pip install -r https://raw.githubusercontent.com/interpretml/interpret-community/master/requirements.txt --user
```
this is just something we use mainly in our tests.  Was there a specific package you had trouble with?

Reply:
Yeah not sure, I think the point was that he wasn't able to use the vis dashboard without the additional pip install lines. I will not approve this merge.

Reply:
Can you try this from your end?
```
conda create -n interpretml python=3.6
conda activate interpretml
pip install azureml-interpret
pip install azureml-contrib-interpret
```
and then run this Python script:
```
from interpret_community.widget import ExplanationDashboard
```

it fails with `ModuleNotFoundError: No module named 'flask'`. Meaning `azureml-interpret` and `azureml-contrib-interpret` are not a complete list.

Reply:
you need to install
```
pip install interpret-community[visualization]
```

Reply:
the visualization extra includes flask et al